diagnostic.md

```diff
@@ -3,7 +3,12 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
+.is-copy-enabled
+.logged-in
+.repository-content
+.commit-tease
+
+all from class
```

Comment:
Not quite. You would need the _entire_ class name in order for the selector to work. For example, there is no class name that is just `logged-in`. There is `logged-in  env-production macintosh vis-public` and `header header-logged-in true`.